docs/security-legal-pii/security/ip-ranges.mdx

```diff
@@ -130,13 +130,26 @@ These IP addresses are only for Sentry use.
 Sentry uses the following IP addresses for uptime checks:
 
 US
+
 ```plaintext
 34.123.33.225
 34.41.121.171
+34.169.179.115
+35.237.134.233
+34.85.249.57
 ```
 
 EU
+
 ```plaintext
 34.159.197.47
 35.242.231.10
+34.107.93.3
+35.204.169.245
 ```
+
+<Alert title="Note" level="warning">
+
+Uptime Monitoring IP addresses may change over time. If you need to programmatically verify whether a visitor is a Sentry bot, we recommend checking the [user agent](/product/alerts/uptime-monitoring/troubleshooting/#user-agent) instead.
+
+</Alert>
```

Comment:
FYI you can write this normally now since we use the new mdx

```suggestion
<Alert title="Note" level="warning">
Uptime Monitoring IP addresses may change over time. If you need to programmatically verify whether a visitor is a Sentry bot, we recommend checking the [user agent](/product/alerts/uptime-monitoring/troubleshooting/#user-agent) instead.
</Alert>
```

Reply:
should work afaik